hack/e2e/eksctl.sh

```diff
@@ -6,7 +6,7 @@ function eksctl_install() {
   INSTALL_PATH=${1}
   EKSCTL_VERSION=${2}
   if [[ ! -e ${INSTALL_PATH}/eksctl ]]; then
-    EKSCTL_DOWNLOAD_URL="https://github.com/weaveworks/eksctl/releases/download/${EKSCTL_VERSION}/eksctl_$(uname -s)_amd64.tar.gz"
+    EKSCTL_DOWNLOAD_URL="https://github.com/weaveworks/eksctl/releases/download/v${EKSCTL_VERSION}/eksctl_$(uname -s)_amd64.tar.gz"
```

Comment:
Should we add 'v' here?

looks like +v is 404 not found --> https://github.com/weaveworks/eksctl/releases/download/v0.11.1/eksctl_Darwin_amd64.tar.gz

vs https://github.com/weaveworks/eksctl/releases/download/0.11.1/eksctl_Darwin_amd64.tar.gz is the correct download link

Reply:
yes, they changed it recently so since we updated to 0.69.0 it's using the v format